README.md

```diff
@@ -17,20 +17,35 @@
 **Release notes and unreleased changes can be found in the [CHANGELOG](CHANGELOG.md)**
 
 ## Requirements
-- rust v1.56 and up
+- Rust v1.56 and up
 - MacOS (x86) or Linux (x86) for sandbox tests. Testnet is available regardless
 
 ### M1 MacOS
-NOTE: Current version of workspaces does not provide support for M1 macbooks, due to some internal upgrades with wasmer. Look at using for workspaces version `0.1.1` for now until we get this settled out or check the progress of it with this [issue](https://github.com/near/workspaces-rs/issues/110).
+NOTE: Current version of `workspaces-rs` does not support use on M1 chip devices due to internal upgrades with wasmer. M1 users should use `workspaces-rs` version `0.1.1` until this problem gets resolved. Check the progress on this issue [here](https://github.com/near/workspaces-rs/issues/110).
 
-To be able to use this library on an M1 Mac, we would need to setup rosetta plus our cross compile target:
+Even with the above note, we can use `workspaces-rs` with version `0.1.1` on M1 by setting up rosetta plus our cross compile target:
 ```
 softwareupdate --install-rosetta
 rustup default stable-x86_64-apple-darwin
 ```
 
-### Note about compiling with contract
-`workspaces-rs` does not currently support compiling to WASM. So, if we are compiling this library alongside a `wasm32` target such as compiling alongside a NEAR contract, it is best if we put this dependency in `[dev-dependencies]` section to avoid any conflicts.
+### Note about compiling with NEAR Contracts
+`workspaces-rs`, the library itself, does not currently compile to WASM. So, if we were compiling it alongside a `wasm32` target, such as compiling alongside a NEAR contract:
+```toml
+# Cargo.toml
+[dependencies]
+workspaces = "*"
```

Comment:
Why are we showing an example of a broken pattern in the readme?

Reply:
Just trying to show the dichotomy of how it'd look like if it didn't work vs the working example. Any suggestions to make it better?

Reply:
This just seems like something relating to the SDK. I feel like if we just share that it does not compile to wasm with some information about that would be sufficient, rather than showing an example of workspaces being included in the SDK dependencies (which it should never be).

Opinionated so nothing has to change, just sharing thoughts that it might be confusing to someone reading

Reply:
I see, yeah that's a good point. I still mentioned the SDK since that seems to be the most common case people would hit from what I've seen. Added the fix here: https://github.com/near/workspaces-rs/pull/115